.github/workflows/check-dependencies.yml

```diff
@@ -3,7 +3,6 @@ name: "3rd-party"
 on:
   push:
     branches:
-      - master
```

Comment:
I'm not sure why need to remove it, since it is a triggering branch

Reply:
> I'm not sure why need to remove it, since it is a triggering branch

When pushing to the master base-ref/head-ref are not automatically assigned, leading to frequent errors during such pushes. The reason for removing them is that during PRs, we can directly check for the introduction of unauthorized 3rd-party libraries (exceptional libraries) through 'Dependency Review' and make the decision on whether to introduce them. There is no need to showcase this during master branch pushes.
![1694346048418](https://github.com/apache/incubator-hugegraph/assets/87920097/bf8ee5e7-6b75-4b77-b5e5-b9342b3f8820)
![1694346099572](https://github.com/apache/incubator-hugegraph/assets/87920097/3cc961b0-4759-41e8-a71b-f534588fde34)

Reply:
Get it, Thanks for the detailed description